programs/vote/src/vote_state.rs

```diff
@@ -423,6 +452,17 @@ pub fn process_vote(
     verify_authorized_signer(&vote_state.authorized_voter, signers)?;
 
     vote_state.process_vote(vote, slot_hashes, clock.epoch)?;
+    if let Some(timestamp) = vote.timestamp {
+        let slot = vote
+            .slots
+            .iter()
+            .max()
+            .expect("vote must contain at least one slot");
+        let res = vote_state.process_timestamp(*slot, timestamp);
+        if let Err(err) = res {
+            warn!("Timestamp: {:?}, Error: {:?}", (slot, timestamp), err);
```

Comment:
I assume we would not want to fail the whole Vote on an invalid timestamp. Correct?
Maybe this should be even quieter (info?)

Reply:
same level of logging as other rejected votes, IMHO

I think I'd want the timestamp to be valid to let in a vote, yes.